dist/Typist.js

```diff
@@ -214,9 +214,10 @@ module.exports =
 	      var isDone = this.state.isDone;
 
 	      var innerTree = utils.extractTreeWithText(this.props.children, this.state.text);
+	      var WrapComponent = this.props.component;
```

Comment:
would you mind removing this generated code from the PR? I'll make sure to generate a new `dist` when I cut a new version.